piop/src/sumcheck/tests/utils.rs

```diff
@@ -0,0 +1,110 @@
+use ark_std::{end_timer, start_timer, vec::Vec};
+use crypto_bigint::Random;
+use crypto_primitives::PrimeField;
+use rand::{Rng, RngCore};
+use std::sync::Arc;
+use zinc_poly::{mle::DenseMultilinearExtension, utils::ArithErrors};
+
+#[allow(clippy::arithmetic_side_effects)]
+#[allow(clippy::type_complexity)]
+pub(crate) fn rand_poly<F: PrimeField + Random, Rn: RngCore>(
+    nv: usize,
+    num_multiplicands_range: (usize, usize),
```

Comment:
This is used only as a half-open range, so just take it as such?
```suggestion
    num_multiplicands_range: Range<usize>,
```